src/SprykerEco/Zed/AfterPay/Business/Mapper/AfterPayMapper.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\AfterPay\Business\Mapper;
+
+use Generated\Shared\Transfer\AfterPayCallTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+
+class AfterPayMapper implements AfterPayMapperInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\AfterPayCallTransfer
+     */
+    public function mapQuoteTransferToAfterPayCallTransfer(QuoteTransfer $quoteTransfer): AfterPayCallTransfer
+    {
+        $afterPayCallTransfer = $this->buildAfterPayCallTransfer($quoteTransfer);
+        $this->addItemsTotalTaxToAfterPayCallTransfer($afterPayCallTransfer, $quoteTransfer);
```

Comment:
$afterPayCallTransfer =  $this->addItemsTotalTaxToAfterPayCallTransfer($afterPayCallTransfer, $quoteTransfer);